packages/keyring-controller/src/KeyringController.ts

```diff
@@ -168,6 +165,41 @@ const defaultState: KeyringControllerState = {
   keyrings: [],
 };
 
+/**
+ * Assert that the given keyring has an exportable
+ * mnemonic.
+ *
+ * @param keyring - The keyring to check
+ * @throws When the keyring does not have a mnemonic
+ */
+export function assertHasUint8ArrayMnemonic(
```

Comment:
Nit: do these need to be exported? They seem more like internal functions - not overly useful for users of this package.

Reply:
They are exported mainly to facilitate testing, otherwise we should manipulate the mnemonic of the internal HD Keyring instance

Perhaps we can move these to a utils.ts file which we won't export from index.ts
Would that work?

Reply:
Hmm. It sounds like you're saying that it's difficult to test this assertion because it's used internally in a code path that isn't reachable. If that's the case, perhaps we should remove the assertion and add a type cast instead, explaining that the HD keyring alway has a UInt8Array mnemonic?

Or if we want to keep the runtime assertion, your solution of moving it to a separate module sounds good.

Reply:
I removed the export from `assertHasUint8ArrayMnemonic` and removed the QR one completely.
I gave coverage to `assertHasUint8ArrayMnemonic` through methods where it is used